packages/common/container-definitions/src/loader.ts

```diff
@@ -207,6 +207,11 @@ export interface IContainer extends IEventProvider<IContainerEvents>, IFluidRout
      */
     readonly isDirty: boolean;
 
+    /**
+     * Disposes the container
```

Comment:
Can we get a little bit more info here to help distinguish from close? (and likewise for close below)